snapshots.go

```diff
@@ -390,6 +394,14 @@ func (s *Stream) createSnapshot(ctx context.Context, dataBuffer, metadataBuffer
 	}
 
 	sub, err := s.mgr.nc.Subscribe(ib, func(m *nats.Msg) {
+		clientInfoHeader := m.Header.Get(server.ClientInfoHdr)
```

Comment:
this will never change so I think I'd rather avoid referencing the server directly here, make a local constant or use a string

Reply:
Added a local constant